package.json

```
@@ -35,6 +35,13 @@
     "core:loaded-shell-environment"
   ],
   "configSchema": {
+    "enable": {
+      "title": "Enable",
+      "description": "Whether to enable this package’s behavior. Provided as a way to disable linting for certain projects without having to disable the package.",
+      "type": "boolean",
+      "default": false,
```

I would prefer this to be `default: true`, as the UX is slightly better 🤔 
```suggestion
      "default": true,
```

---

Whoops — why on earth did I default this to `false`? Good catch.